omod/src/main/resources/config.xml

```diff
@@ -8,7 +8,7 @@
 	<name>@MODULE_NAME@</name>
 	<version>@MODULE_VERSION@</version>
 	<package>@MODULE_PACKAGE@</package>
-	<author>Viet Nguyen, Chuyen Nguyen</author>
+	<author>ricky</author>

```

Comment:
Author should be **HealthIT** NOT individual names

Reply:
Done